tests/integration/mock/eventstream_runtime/CodeActAgent/test_browse_internet/prompt_001.log

```diff
@@ -402,7 +402,13 @@ The server is running on port 5000 with PID 126. You can access the list of numb
 
 
 NOW, LET'S START!
+
+----------
```

Comment:
Just curious, why does it do a separator now and it didn't before? 🤔

Reply:
Weird, text was a string, now it's a string, there was no conversion or something 😅